userguide/system.rst

```diff
@@ -2024,52 +2028,52 @@ as the signing authority.
    +-------------------------+----------------------+-------------------------------------------------------------------------------------------------+
-   | Key Length              | drop-down menu       | for security reasons, a minimum of *2048* is recommended                                        |
+   | Key Length              | drop-down menu       | For security reasons, a minimum of *2048* is recommended.                                       |
    |                         |                      |                                                                                                 |
    +-------------------------+----------------------+-------------------------------------------------------------------------------------------------+
-   | Digest Algorithm        | drop-down menu       | the default is acceptable unless your organization requires a different algorithm               |
+   | Digest Algorithm        | drop-down menu       | The default is acceptable unless the organization requires a different algorithm.               |
    |                         |                      |                                                                                                 |
    +-------------------------+----------------------+-------------------------------------------------------------------------------------------------+
-   | Lifetime                | integer              | in days                                                                                         |
+   | Lifetime                | integer              | The lifetime of the cetificate is specified in days.                                            |
    |                         |                      |                                                                                                 |
    +-------------------------+----------------------+-------------------------------------------------------------------------------------------------+
-   | Country                 | drop-down menu       | select the country for the organization                                                         |
+   | Country                 | drop-down menu       | Select the country for the organization.                                                        |
    |                         |                      |                                                                                                 |
    +-------------------------+----------------------+-------------------------------------------------------------------------------------------------+
-   | State                   | string               | required; enter the state or province for the organization                                      |
+   | State                   | string               | Enter the state or province for the organization.                                               |
```

Comment:
On text fields like these, "Enter" is not needed.  Also, it's not really a field "for" the organization, but "of" it:

State or province of the organization.